tests/Unit/SDK/Resource/ResourceTest.php

```diff
@@ -241,6 +243,32 @@ public function test_merge(): void
         $this->assertEquals('', $empty);
     }
 
+    /**
+     * @dataProvider schemaUrlsToMergeProvider
+     */
+    public function test_merge_schema_url($oldScheme, $updatingScheme, $expectedScheme, $shouldLog): void
+    {
+        if ($shouldLog) {
```

Comment:
Please, do not use conditions in tests. This can simply be a separate test.

Reply:
Good point, thanks!